src/tag-tree-data-provider.ts

```diff
@@ -1,6 +1,5 @@
 import { debounce } from "debounce";
 import * as fs from "fs";
-import * as recursiveReadDir from "recursive-readdir";
```

Comment:
We'll need to pull this out of package.json but I'm not going to block your pr because of that.

Reply:
Good call, I missed that.